modules/@apostrophecms/modal/ui/apos/components/AposDocsManagerToolbar.vue

```diff
@@ -91,7 +108,7 @@ export default {
       searchField: {
         field: {
           name: 'search',
-          placeholder: `Search ${this.labels.plural}`,
+          placeholder: `Search ${this.labels.plural || ''}`,
```

Comment:
the pages manager doesn't have search, so there's no other reason to be including the labels property.